binaries/workerserver/main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	handler := server.NewHandler(stat.Scope("workerserver"))
+	handler := server.DefaultHandler(stat.Scope("workerserver"))
```

Comment:
Any chance I could convince you to undo DefaultHandler and require injecting the runner dependency? I know it looks a little worse right here, but I think it's really good if you have to pass the Runner, because it means that when you're looking at the workerserver, you know it's running something.

(It also means that we don't need to agonize over what the default should be)